src/components/standalone-link/index.tsx

```diff
@@ -68,9 +80,9 @@ const StandaloneLink = ({
 			opensInNewTab={opensInNewTab}
 		>
 			<StandaloneLinkContents
-				color={color}
 				icon={icon}
 				iconPosition={iconPosition}
+				inheritColor
```

Comment:
It is taking me a little while to understand this `inheritColor` behavior...

It appears that on the `StandaloneLinkContents`, setting `inheritColor` simply just disallows the usage of the `color` attribute.  Are there any other behaviors here that I am missing?

It also appears that the only place that we would ever use `inheritColor=true` is on this line here (to inherit the color as set from the `StandaloneLink`).

If those is correct, could we simply just remove this `inheritColor` prop and simply just _not_ pass a color to the `StandaloneLinkContents`?

Reply:
@BrandonRomano this section of code already doesn't pass `color` down to `StandaloneLinkContents`. Because `StandaloneLinkContents` has a `color` prop, and by default applies the helper classes based on that prop, `StandaloneLink` needs a way to disable that behavior so it can handle the color setting _for_ `StandaloneLinkContents`.

Here's an example of how `StandaloneLinkContents` will be used on the home page:
https://github.com/hashicorp/dev-portal/pull/1823/files#diff-4cefbd1b2095426710643ee7c5e1f98fa0ebda085a4ce03b49e76b175b5f0f71R67-R72

I basically need all of `StandaloneLink`'s behavior and styling, aside from a link being rendered and color changing on active/hover. So I need `color`, `icon`, `iconPosition`, and `text` handling here, and in the other featured cards in this section of the page those props and `size` will be useful.

Reply:
@BrandonRomano @ashleemboyer If I'm following Brandon's suggestion, I think an alternative to the `inheritColor` prop could be to _remove_ the `DEFAULT_COLOR_VARIANT` fallback from `StandaloneLinkContents`. Then we could not pass `color` when rendering `StandaloneLinkContents` in `StandaloneLink`. l think that's a pretty good idea, and could remove some props complexity 👍

I think the tradeoff would be it'd then be maybe easier to "misuse" `StandaloneLinkContents` - it'd be possible to render that component without a `color` prop, resulting in unexpected but difficult-to-warn-against appearance.

Personally happy with either solution 🌻